apps/web/hooks/escrow/use-Escrow.ts

```
@@ -0,0 +1,131 @@
+      const records = data as EscrowRecord[]
+      const currentRecord = records.find(record => record.id === escrowId)
+      setEscrowData(currentRecord || null)
+    } catch (err) {
+      const errorMessage = handleError(err)
+      setError(errorMessage)
+    } finally {
+      setIsLoading(false)
+    }
+  },   [escrowId, handleError])
+
+  const updateStatus = useCallback(async (newStatus: EscrowStatusType): Promise<EscrowResponse> => {
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: handleError

This dependency is not specified in the hook dependency list.


(lint/correctness/useExhaustiveDependencies)